core/sdk/src/framework/ripple_contract.rs

```diff
@@ -41,12 +41,7 @@ pub enum RippleContract {
     /// Used by Main application to provide internal contracts for Extensions
     #[default]
     Internal,
-    /// Provided by the distributor useful for adding Governance implementation for handling
-    /// privacy information and other sensitive data.
```

Comment:
Curious why we're removing contract testing for these? Or are the contract enums gong to be defined elsewhere?

Reply:
We are removing contracts which are no longer necessary for our implementation